### ISSUE TITLE
1.3.0(313) Alert range is reset on cloud sync

### ISSUE THREAD

**Issue by markoaamunkajo** [ruuvi/com.ruuvi.station.ios#1045]
Description:

BT Alert is reset upon cloud sync

User is logged into Ruuvi Cloud and using app in mixed mode (local bt and cloud).
User sets humidity alert and threshold so the alert would trigger, but actually the alert doesn't trigger with local Bluetooth. He then waits for synchronizing... text on sensor card and afterwards sees that humidity alert threshold range has been reset in sensor settings page.

https://user-images.githubusercontent.com/50437378/182067626-60c0da43-fa00-43b7-b7c2-418975b27eef.mov

**Comment by markoaamunkajo** [comment]
Gave more details about this case in dm will move back to @priyonto

**Comment by priyonto** [comment]
I have improved a few related pages to this on build 314. However, I am not expecting magical improvement either. As we have discussed, the behavior is expected considering how the cloud sync is being performed at this moment. While in normal use cases the issue should not be critical, in cases like slow and unpredictable networks the changes may be lost. 

The implementation of #746 should eliminate this issue and #1055 should improve the experience overall.
